deploy/kik-testnet/datadog-agent/etc/datadog-agent/conf.d/json_check.yaml

```diff
@@ -1,7 +1,172 @@
+#    metrics:
+#      - archive.latest_ledger: currentLedger # Latest ledger updated in the history archive
+
+#Horizon metrics:
+# - url: https://horizon.stellar.org/metrics
+#   metrics:
+#   # Metrics that include "5_min_rate" report the 5 minute moving avg pre second of this metric
+#     - horizon.goroutines: goroutines.value # goroutines used by the horizon app
+#     - horizon.elder_ledger: '"history.elder_ledger".value' # oldest ledger in the horizon db
+#     - horizon.latest_ledger: '"history.latest_ledger".value' # ewest ledger in the horizon db
+#     - horizon.open_connections: '"history.open_connections".value' # open connections to the horizon database
+#     - horizon.logging_error: '"logging.error"."5_min_rate"' # error logs
+#     - horizon.logging_panic: '"logging.panic"."5_min_rate"' # error logs
+#     - horizon.logging_warning: '"logging.warning"."5_min_rate"' # warning logs
+#     - horizon.requests_failed: '"requests.failed"."5_min_rate"' # failed requests to horizon (400<= status code < 600)
+#     - horizon.requests_succeeded: '"requests.succeeded"."5_min_rate"' # successful requests to horizon (200<= status code < 400)
+#     - horizon.requests_total: '"requests.total"."5_min_rate"' # total requests to this horizon
+#     - horizon.core_open_connections: '"stellar_core.latest_ledger".value' # open connections to the stellar core database
+#     - horizon.txsub_buffered: '"txsub.buffered".value' # submission buffered in the submission queue
+#     - horizon.txsub_failed: '"txsub.failed"."5_min_rate"' # failed transactions sent from this proccess
+#     - horizon.txsub_open: '"txsub.open".value' # transactions that have yet to be confirmed as successful or failed
+#     - horizon.txsub_succeeded: '"txsub.succeeded"."5_min_rate"' # successfull transactions sent from this proccess
+#     - horizon.txsub_total: '"txsub.total"."5_min_rate"' # total number of transactions sent to this proccess
+
+#Core metrics:
+# - url: http://localhost:11626/metrics
+#   metrics:
+#   # All metrics report the 5 minute moving avg per second of this metric
+#     # not sure what this is
+#     - core.crypto_verify_hit: 'metrics."crypto.verify.hit"."5_min_rate"'
+#     - core.crypto_verify_miss: 'metrics."crypto.verify.miss"."5_min_rate"'
+#     - core.crypto_verify_total: 'metrics."crypto.verify.total"."5_min_rate"'
+#     # herder metrics are related to implementaion of the SCP
+#     - core.herder_pending-txs_age0: 'metrics."herder.pending-txs.age0".count'
+#     - core.herder_pending-txs_age1: 'metrics."herder.pending-txs.age1".count'
+#     - core.herder_pending-txs_age2: 'metrics."herder.pending-txs.age2".count'
+#     - core.herder_pending-txs_age3: 'metrics."herder.pending-txs.age3".count'
+#     - core.herder_state_changes: 'metrics."herder.state.changes"."5_min_rate"'
+#     - core.herder_state_current: 'metrics."herder.state.current".count'
+#     # metrics regarding the history archives
```

Comment:
put a line space between each block here for better readability, it's hard to differentiate between metric names and comments